src/Config/Configuration.php

```diff
@@ -29,7 +29,7 @@ public function __construct(array $settings = [])
     /**
      * Return a setting by dot.separated.syntax
      * @param string|array $path
-     * @param mixed $default
+     * @param ?mixed $default
```

Comment:
I think the PHPDoc way of declaring nullability is to use unions (i.e. `mixed|null`) - but in any case, [mixed is inherently nulalble.](https://wiki.php.net/rfc/mixed_type_v2#nullability)
```suggestion
     * @param mixed $default
```

Reply:
`get()` doesn't seem to be declared higher in the inheritance than this (i.e. it's not in the trait) - but I'm guessing this wasn't typehinted for PHP7.4 compatability?

Reply:
Are you asking why I didn't add a explicit `:mixed` return type?

`mixed` doesn't exists in PHP7.4.

Reply:
Yup, and yeah that's what I thought was probably the case. No issue there then :) Just gotta fix the phpdoc from`?mixed` to `mixed`